src/plone/restapi/services/users/add.py

```diff
@@ -33,6 +36,24 @@ def publishTraverse(self, request, name):
         self.params.append(name)
         return self
 
+    def translate(self, msgid, domain="plone.restapi"):
+        return translate(
+            msgid,
+            domain=domain,
+            context=self.request,
+        )
+
+    def translate_fieldname(self, fieldname):
+        """Not all fields that appear on Add User Form in Volto match in Plone."""
+        msgid = ""
+        if fieldname == "roles":
+            msgid = "Roles"
+        elif fieldname == "sendPasswordReset":
+            msgid = ICombinedRegisterSchema["mail_me"].title
+        else:
+            msgid = ICombinedRegisterSchema[fieldname].title
+        return self.translate(msgid, "plone")
```

Comment:
@wesleybl I think it might be better to _not_ pass a domain when translating the field title. Zope message ids have an attached domain (inherited from the message factory used to construct it) and zope.i18n.translate will use it unless a different domain is passed in: https://github.com/zopefoundation/zope.i18n/blob/master/src/zope/i18n/__init__.py#L166

Reply:
@davisagli the problem is the Roles field up there:

https://github.com/plone/plone.restapi/blob/da4ebc95a5eb25c261a0bb7cbedca3c521527a5f/src/plone/restapi/services/users/add.py#L49-L50
The `ICombinedRegisterSchema` interface does not have a Roles field. An interface that has this field is `IRoleCondition`:

https://github.com/plone/plone.app.contentrules/blob/4d3e8e28eb4c8a1d0cfed36d9f4543338c94f470/plone/app/contentrules/conditions/role.py#L23-L28

But I don't know if it makes sense to import it here.

Another option would be to create an entry in the `plone.restapi` domain:

```
_("Roles")
```
What would be the best option here?

Reply:
Well, I thought it best to put the Roles in the plone.restapi domain.

Thus, I no longer pass the domain.